sentry.properties

```diff
@@ -0,0 +1,3 @@
+defaults.project=android
+defaults.org=man
+auth.token=
```

Comment:
@bruno-garcia we might need a take token where the server will not refuse it

Reply:
We wouldn't want to check-in an auth token though. I'm not sure we could limit it only for uploading proguard files so.
We can set this via environment variable, and `sentry-cli` will pick it up

Reply:
ok I kept it commented for now and let's discuss it later on